ios/Podfile.lock

```diff
@@ -106,4 +106,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: c34e2287a9ccaa606aeceab922830efb9a6ff69a
 
-COCOAPODS: 1.9.1
+COCOAPODS: 1.8.4
```

Comment:
Any reason we downgrade cocoapods version? Maybe running locally changed it by accident.

Reply:
@yimingc2 I'll do a pod update and re push.

Reply:
@workerbee22 I think your local cocoapods version is 1.8.4. At the moment, we don't have Gemfile to let everyone run the same cocoapods version. So I think we can simply ignore it for now. If we don't need to update any dependencies, then we don't need to `pod update`. If someone just downloads the app and start from stretch, he can use `pod install` to download the required version specified in Podfile.lock

Reply:
Fixed with last commit. Now 1.9.1 👍